src/sentry/api/serializers/models/group.py

```diff
@@ -713,21 +762,41 @@ def __init__(
         self.stats_period_end = stats_period_end
         self.matching_event_id = matching_event_id
 
-    def query_tsdb(self, group_ids, query_params):
+    def _get_seen_stats(self, item_list, user):
+        partial_execute_seen_stats_query = functools.partial(
+            self._execute_seen_stats_query,
+            item_list=item_list,
+            environment_ids=self.environment_ids,
+            start=self.start,
+            end=self.end,
+        )
+        filtered_result = partial_execute_seen_stats_query(conditions=self.conditions)
+        time_range_result = partial_execute_seen_stats_query()
+        lifetime_result = partial_execute_seen_stats_query(start=None, end=None)
```

Comment:
I actually didn't realise you could override the values of already passed args in a partial like this and had to verify, looks good!

Reply:
Haha yeah, me too. Liked this better than passing start + end to filtered + time_range :-)